conf/config-default.yaml

```diff
@@ -101,15 +101,14 @@ apisix:
   dns_resolver_valid: 30          # valid time for dns result 30 seconds
   resolver_timeout: 5             # resolver timeout
   ssl:
-    enable: false                 # ssl is disabled by default
-                                  # enable it to use your own cert and key
+    enable: true
     enable_http2: true
     listen_port: 9443
     # ssl_trusted_certificate: /path/to/ca-cert # Specifies a file path with trusted CA certificates in the PEM format
                                                 # used to verify the certificate when APISIX needs to do SSL/TLS handshaking
                                                 # with external services (e.g. etcd)
-    # ssl_cert: /path/to/ssl_cert
-    # ssl_cert_key: /path/to/ssl_cert_key
+    ssl_cert: "cert/ssl_PLACE_HOLDER.crt"
```

Comment:
I think we can remove this option, the user does not need to care about it

Reply:
It's good to save this, since sometimes users may need to customize the default certs.

Reply:
this default cert is useless. and the APISIX never uses those certs.

Reply:
Thanks! PTAL